app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -1,13 +1,46 @@
+import * as R from "ramda";
 import * as yup from "yup";
 
+export const CONTACTS = [
+  { value: 1, label: "Ronald Richards" },
+  { value: 2, label: "Jacob Jones" },
+];
+
+export const TAGS = [
+  { value: 1, label: "Getting Started" },
+  { value: 2, label: "Onboarding" },
+  { value: 2, label: "User flow" },
+  { value: 2, label: "Ux" },
+  { value: 2, label: "Bugs" },
```

Comment:
Why are all the other tags except first one having value as `2`?

Reply:
Will change it asap.